projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -276,3 +275,17 @@ <h3 id="{{ 'doc-altgr-title-' + i }}">{{ altgr_title }}</h3>
     [resourcePid]="record.metadata.pid"
   ></admin-operation-logs-dialog>
 </ng-container>
+
+<ng-template #entityLink let-entity="entity" let-class="class">
+  <i class="fa fa-tag mr-1"></i>
+  <a
+    [class]="class"
+    *ngIf="entity.entity.resource_type; else subjectNoLink"
```

Comment:
is `*ngIf` could be the first tag attribute (before `[class]`) to increase readability